bitrise.yml

```diff
@@ -2123,13 +2128,15 @@ app:
 
 trigger_map:
 - push_branch: main
-  pipeline: pipeline_build_and_test
+  workflow: L10nBuild
```

Comment:
This is the change you have to do to try a workflow on Bitrise. The trigger specifies the workflow. Since the L10nBuild workflow does not have dependencies, it works by itself when it is triggered directly.

Reply:
Once we have tested that this works, we should set the pipeline back by removing this and uncommenting the below line